documentation/general/contributing.md

```diff
@@ -175,3 +178,146 @@ If you know folks who have expertise in any of the above areas who you think
 might be interested in contributing to open source, send them our way! We're
 happy to help onboard folks to the project itself, as well as the tools and
 technologies we use.
+
+### Issue backlog maintenance
+
+The [Openverse issue backlog](https://github.com/WordPress/openverse/issues)
+requires regular maintenance and upkeep. Issues can become stale or unworkable
+for a variety of reasons. For example, they can:
+
+- be duplicates of other issues
+- have been silently fixed
+- have been explicitly fixed but accidentally left open
+- require additional context before they can be worked on
+- refer to implementation details that are no longer relevant due to
+  infrastructural or design changes
+- the prioritisation of an issue could need changing
+
+These conditions cannot be easily identified automatically and require regular,
+ongoing work from contributors. Additionally, important issues can be
+accidentally ignored dur to the breadth of our backlog and it is critical for
+maintainers to dig through the oldest issues to ensure that issue prioritisation
+still makes sense.
+
+The following sections give hints on how to manage certain common scenarios.
+Generally speaking, for any scenario where an issue is being closed, it is good
+to ping at least one other maintainer for advice or to corroborate your
+understanding of the situation. If you find issues that don't seem right for any
+reason, [ping any of the communication aliases](/meta/communication_aliases.md)
+and someone will help clear up any doubts.
+
+#### Useful issue query links
+
+- [Ticket work required](https://github.com/WordPress/openverse/labels/%F0%9F%A7%B9%20status%3A%20ticket%20work%20required);
+  see [the section below](#issues-that-need-more-information) for actionability
+- [Medium and low priority sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+label%3A%22%F0%9F%9F%A8+priority%3A+medium%22%2C%22%F0%9F%9F%A9+priority%3A+low%22+sort%3Acreated-asc);
+  see [the section below](#updating-prioritisation) for actionability
+- [All issues sorted by oldest first](https://github.com/WordPress/openverse/issues?q=is%3Aopen+is%3Aissue+sort%3Acreated-asc+)
+
+#### Issues that need more information
+
+Issues may not have all the context required for them to be worked on. This can
+be caused by someone opening issues in a hurry, by the context around an issue
+changing over time, by links meant to clarify the context rotting, or other
```

Comment:
```diff
- by links meant to clarify the context rotting
+ by outdated or broken links
```

might be easier to understand. "Link rot" is a little technical and splitting it with "meant to clarify the context" it gets even more unclear.

other solutions welcome!